README.md

```diff
@@ -252,6 +252,27 @@ Remember to include all databases used by
 [edx-platform](https://github.com/openedx/edx-platform), as well as
 those created by any plugins installed.
 
+### Selecting the MongoDB databases to backup
+
+By default, only the MongoDB database set in Tutor's `MONGODB_DATABASE`
```

Comment:
Hmmm. Looking at how this is implemented, I now *think* that if we do it this way, then this would constitute a breaking change. Someone who is using this plugin to backup a MongoDB server that does have a `cs_comments_service` database would expect it to be included in the backup, but their backups would suddenly no longer include that database after upgrading the plugin (and leaving the configuration unchanged).

How about we make `BACKUP_MONGODB_DATABASES` an empty list by default instead, and only back up select databases if the user overrides the default? That would also make this option's behavior more like that of `BACKUP_MYSQL_DATABASES`.

Reply:
I see your point. The downside of the approach you suggest is that we would dump and restore the `admin` database if `BACKUP_MONGODB_DATABASES` is not provided. That would put the responsibility on the user to provide `BACKUP_MONGODB_DATABASES` if they don't want their `admin` database to be overwritten. I am ok with doing it that way if you think that is an accepted behavior. It will definitely simplify the implementation.

Reply:
Well actually now that I think of it, we have already put the responsibility on the user in the case of `BACKUP_MYSQL_DATABASES`. So doing it for MongoDB should be OK as well. I'll change the implementation.